dataprofiler/tests/data_readers/test_text_data.py

```diff
@@ -25,27 +26,56 @@ def setUpClass(cls):
         ]
         cls.output_file_path = None
 
+        cls.buffer_list = []
```

Comment:
i have one concern, are any of the above not utf-8 / ascii?

Reply:
Maybe should be in a follow-up PR to fix as I can see this having issues if encoding is not detected and appropriately selected.